runtime/oti/j9nonbuilder.h

```diff
@@ -5286,6 +5286,9 @@ typedef struct J9JavaVM {
 	struct J9ClassLoader* anonClassLoader;
 	UDATA doPrivilegedWithContextPermissionMethodID1;
 	UDATA doPrivilegedWithContextPermissionMethodID2;
+#if JAVA_SPEC_VERSION >= 15
```

Comment:
This needs to be unifdeffed as it is used unconditionally in the JIT code.

Reply:
Right, removed the `#if` here, also set `vm->nativeLibrariesLoadMethodID = 0` for pre-JDK15 levels.

Reply:
FYI - the J9JavaVM is zeroed on allocation.